src/Dropdown/DropdownListTrigger.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import PropTypes from "prop-types";
+import DropdownTrigger from "./DropdownTrigger";
+
+export default class DropdownListTrigger extends DropdownTrigger {
+  render() {
+    const { className, disabled, selectedItem } = this.props;
+    const html = selectedItem
+      ? selectedItem.selectedHtml || selectedItem.html
+      : this.props.placeholder || null;
+
+    return (
+      <button
+        className={className}
+        disabled={disabled}
+        onClick={this.handleTrigger}
```

Comment:
Nit: shouldn't this be `super.handleTrigger`?

Reply:
Hm, I think not, imagine something like this:
```
class ExtendedDropdownListTrigger extends DropdownListTrigger {
  handleTrigger(event) {
    super.handleTrigger(event);
    SomeExtendedFunctionality.onDropdownTrigger(event)
  }
}
```
=> I'm not aware of calling super from other methods then the overwritten one 🤔

Reply:
It can be used outside of same method calls...see example of class Rectangle in the link: https://googlechrome.github.io/samples/classes-es6/

Reply:
sure it _can_ be used anywhere, but is it useful/necessary? serious question, if you dont explicitly want to prevent further extension, what is the advantage of calling `super.something()` instead of `this.something()`?

Reply:
It doesn't really matter, you can leave it. Was just a nit-pick.

Reply:
I think the advantage is to skip one prototype jump. Instead of forcing Javascript to look at `this` first before going up in the prototype chain we could point to `super` directly @orlandohohmeier